functions/gameFunctions.go

```diff
@@ -0,0 +1,100 @@
+package functions
+
+import (
+	. "../models"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"sort"
+	"time"
+)
+
+const (
+	expressionFile = "jsons/expressions.json"
+	ratingFile     = "jsons/rating.json"
+)
+
+func StartGame() {
+	var numberCorrectAnswers int
+	var numberIncorrectAnswers int
+	go calculateExpressions(&numberCorrectAnswers, &numberIncorrectAnswers)
+	time.Sleep(time.Minute)
```

Comment:
bad solution
I think better to use WaitGroup for flow control (and move time management into goroutine)